src/System.IO/tests/ConfigurablePropertyStream.cs

```diff
@@ -0,0 +1,62 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.IO.Tests
+{
+    public class ConfigurablePropertyStream : Stream
```

Comment:
Yep; thanks for telling me about that. I've also opted to put `CallTrackingStream` in `Common` as well.